src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  color: $mainAccent;
+  height: 406px;
```

Comment:
instead setting horizontal paddings/margins for each card element set paddings for .card
it will be something like that
![Screenshot from 2023-04-30 18-00-01](https://user-images.githubusercontent.com/33029864/235360124-752091d5-2b52-4701-96f6-72489b19c818.png)